dev/tasks/r/github.macos.autobrew.yml

```diff
@@ -16,44 +16,46 @@
 # under the License.
 
 {% import 'macros.jinja' as macros with context %}
+{% set is_fork = macros.is_fork %}
 
 {{ macros.github_header() }}
 
 jobs:
   autobrew:
-    name: "Autobrew"
-    runs-on: macOS-10.15
+    name: "Autobrew {{ "${{ matrix.platform }}" }}"
+    runs-on: {{ "${{ matrix.platform }}" }}
+    strategy:
+      fail-fast: false
+      matrix:
+          platform:
+            - macos-11
+            - macos-10.15
```

Comment:
Can we remove this becuase `macros-10.15` will be removed soon?

Reply:
Then, could you remove `macos-10.15` in this pull request?
Jobs that use `macos-10.15` may be canceled like https://github.com/ursacomputing/crossbow/actions/runs/2789412814 . It will be shown as a failure in nightly build reports.